.github/workflows/update-watcher.yml

```diff
@@ -16,13 +16,13 @@ jobs:
           node-version: 18
       - run: cd example && npm install --save-dev @axe-core/watcher@next
       - name: Open PR
-        uses: peter-evans/create-pull-request@v3
+        uses: peter-evans/create-pull-request@153407881ec5c347639a548ade7d8ad1d6740e38 # tag=v5
```

Comment:
`.github/workflows/test.yml` and `/action.yml` both need similar updates to pin dependencies